cockatrice/src/game/zones/view_zone_widget.cpp

```diff
@@ -184,25 +215,28 @@ void ZoneViewWidget::retranslateUi()
 {
     setWindowTitle(zone->getTranslatedName(false, CaseNominative));
 
-    { // We can't change the strings after they're put into the QComboBox, so this is our workaround
-        int oldIndex = sortBySelector.currentIndex();
-        sortBySelector.clear();
-        sortBySelector.addItem(tr("Sort by ---"), CardList::NoSort);
-        sortBySelector.addItem(tr("Sort by Name"), CardList::SortByName);
-        sortBySelector.addItem(tr("Sort by Type"), CardList::SortByType);
-        sortBySelector.addItem(tr("Sort by Mana Value"), CardList::SortByManaValue);
-        sortBySelector.setCurrentIndex(oldIndex);
-    }
+    groupByLabel.setText(tr("Group by"));
+    sortByLabel.setText(tr("Sort by"));
 
-    {
+    { // We can't change the strings after they're put into the QComboBox, so this is our workaround
         int oldIndex = groupBySelector.currentIndex();
         groupBySelector.clear();
-        groupBySelector.addItem(tr("Group by ---"), CardList::NoSort);
-        groupBySelector.addItem(tr("Group by Type"), CardList::SortByType);
-        groupBySelector.addItem(tr("Group by Mana Value"), CardList::SortByManaValue);
+        groupBySelector.addItem(tr("---"), CardList::NoSort);
```

Comment:
"---" does not need translations. 

You did not want to put a text label like "None"?

Reply:
I agree --- does not properly convey that this is the deck's own order, perhaps have it say "unsorted" or "sort by position in deck"?